index.rst

```diff
@@ -21,3 +20,16 @@ This tutorial was last given at SciPy 2020 in Austin Texas.
    03_dask.delayed
    04_distributed
    05_futures
+
+
+More tutorials from our community
+---------------------------------
+
+- Coiled offers recurring free 1 hour-long tutorials. See the `Coiled website <https://www.coiled.io/tutorials>`_ for upcoming sessions.
+- `Quansight <https://quansight.com/training>`_ offers a number of PyData courses, including Dask and Dask-ML.
```

Comment:
Small style nit, but I wouldn't start this sentence with the word "Coiled".  Coiled is an afterthought here I think.

Instead I think that we lead with the user, saying something like... "You may want to check out these recurring 1 hour long tutorials [here]() offered by Coiled".

Reply:
Done!